components/gateway/internal/auditlog/client.go

```diff
@@ -29,28 +29,28 @@ type TimeService interface {
 	Now() time.Time
 }
 
+//go:generate mockery -name=HttpClient -output=automock -outpkg=automock -case=underscore
+type HttpClient interface {
+	Do(request *http.Request) (*http.Response, error)
+}
+
 type AuditlogConfig struct {
-	User                 string `envconfig:"APP_AUDITLOG_USER"`
-	Password             string `envconfig:"APP_AUDITLOG_PASSWORD"`
-	AuditLogURL          string `envconfig:"APP_AUDITLOG_URL"`
-	Tenant               string `envconfig:"APP_AUDITLOG_TENANT"`
-	AuditlogConfigPath   string `envconfig:"APP_AUDITLOG_CONFIG_PATH,default=/audit-log/v2/configuration-changes"`
-	AuditlogSecurityPath string `envconfig:"APP_AUDITLOG_SECURITY_PATH,default=/audit-log/v2/security-events"`
+	AuditLogURL          string   `envconfig:"APP_AUDITLOG_URL"`
+	AuditlogConfigPath   string   `envconfig:"APP_AUDITLOG_CONFIG_PATH,default=/audit-log/v2/configuration-changes"`
```

Comment:
these default values are really enigmatic

Reply:
I want to discriminate those 2 services.
According to this comment:
https://github.com/kyma-incubator/compass/pull/975#discussion_r393465002